### DISCUSSION TITLE
Getting Error Page Load ---waiting for new page to load ---

### DISCUSSION THREAD

**Discussion by Twinkle-bonrepublic** [cypress-io/cypress#30072] [Questions and Help]
We are currently implementing a test case which involves cross-origin navigation. Despite trying various approaches, we are facing issues with Cypress where the URL does not change as expected after clicking the SSO-login button. The test breaks while waiting for the page to load, although the page appears to be loaded. Also we have other case running on same origin. Using approaches like 

experimentalSessionAndOrigin: true
chromeWebSecurity: false

<img width="1355" alt="Screenshot 2024-08-19 at 14 34 27" src="https://github.com/user-attachments/assets/1539bdd3-ef8e-4d43-b706-ee136c790dc4">

Sample code 

cy.visit('https://xyz', {
  timeout: 120000,
})

cy.get('button.Button.Button_secondary.SSO_button#HRworksLogin')
  .should('be.visible')
  .and('exist')

cy.get('a[href="https://abc"]')
  .should('be.visible')
  .click()

**Comment by MikeMcC399** [comment]
@tg-bonrepublic 

Have you referred to https://docs.cypress.io/guides/guides/cross-origin-testing and are you using `cy.origin()`?

You can make posted code more readable by using fenced code blocks. See https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks

**Reply by Twinkle-bonrepublic** [reply]
I tried using cy.origin/intercept as well but it doesn't seems to work

**Reply by MikeMcC399** [reply]
Unfortunately I can't help you in more detail. For "how-to" type support you can also connect to the Cypress technical community on Discord

[![Discord chat](https://img.shields.io/badge/chat-on%20Discord-brightgreen)](https://on.cypress.io/discord) (click on button)

You might ask there if you don't get more help here.
